### PR TITLE
Glue is trusted

### LINKED ISSUE #4091: Add AWS Glue adapter to trusted list
### Link to adapter repo

https://github.com/aws-samples/dbt-glue

### Contact Details

mehdmldj@amazon.com

### Which of these best describes you?

I work for the vendor on top of which the dbt adapter functions

### Please agree to the each of the following

- [X] I am a maintainer of the adapter being submited for Trusted status
- [X] I have read both the [Trusted adapters](https://docs.getdbt.com/docs/trusted-adapters) and [Building a Trusted Adapter](https://docs.getdbt.com/guides/dbt-ecosystem/adapter-development/8-building-a-trusted-adapter) pages.
- [X] I believe that the adapter currently meets the expectations given above
- [X] I will ensure this adapter stays in compliance with the guidelines
- [X] I understand that dbt Labs reserves the right to remove an adapter from the trusted adapter list at any time, should any of the below guidelines not be met

### What icon should be used?

image

### PR DESCRIPTION
**Pull request by dataders**
## What are you changing in this pull request and why?

resolves: #4091 

## Checklist

- [ ] dbt-glue repo has PR CI checks enabled by "ok to test" label
- [ ] all tests available in dbt Labs provided suite are present, with those don't pass commented out

### CODE REVIEW

File: website/snippets/_adapters-trusted.md [hunk 1]
```diff
@@ -5,6 +5,11 @@
     body="<a href='/docs/core/connect-data-platform/athena-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install with dbt Core </a> <br /><br /><a href=https://badge.fury.io/py/dbt-athena-community><img src=https://badge.fury.io/py/dbt-athena-community.svg/></a>"
     icon="athena"/>
 
+<Card
+    title="Glue*"
+    body="<a href='/docs/core/connect-data-platform/athena-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install with dbt Core </a> <br /><br /><a href=https://badge.fury.io/py/dbt-glue><img src=https://badge.fury.io/py/dbt-glue.svg/></a>"
```
Comment (mirnawong1) [inline website/snippets/_adapters-trusted.md]: @dataders is the athena link the right one? or should it be the glue-setup one?

```suggestion
    body="<a href='/docs/core/connect-data-platform/glue-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install with dbt Core </a> <br /><br /><a href=https://badge.fury.io/py/dbt-glue><img src=https://badge.fury.io/py/dbt-glue.svg/></a>"
```

**Review by mirnawong1 — CHANGES_REQUESTED** [review]
hey @dataders , left a comment regarding the link. also since the trusted platforms are for dbt core, do you think they should have the asterisk? trying to understand what it's use is for.

File: website/snippets/_adapters-trusted.md (hunk 1 above)
Comment (dataders) [reply website/snippets/_adapters-trusted.md]: good catch!!

**Comment by dataders** [comment]
> hey @dataders , left a comment regarding the link. also since the trusted platforms are for dbt core, do you think they should have the asterisk? trying to understand what it's use is for.

Are you saying that the asterisk footnote saying that "this adapter is only for core" is redundant given that Trusted adapters are Core only?

image

**Comment by mirnawong1** [comment]
that's right @dataders ! it looks like the footnote was removed already though in the live page for [trusted](https://docs.getdbt.com/docs/trusted-adapters) and verified adapters? but other pages have the asterisks so its a little inconsistent - I've removed the asterisks and footnote - i hope that's ok? let me know if you prefer the footnote or asterisks. 

i have approved this for you to merge. 
 
Screenshot 2023-11-16 at 09 46 04

**Review by mirnawong1 — APPROVED** [review]
lgtm @dataders !

**Comment by mirnawong1** [comment]
going to merge this @dataders since it looks like it's final! however, please let me know if you disagree with anything!

**Pull request merged**